connectors/src/resources/db/index.ts

```diff
@@ -0,0 +1,39 @@
+import { eq, or } from "drizzle-orm";
+import { drizzle } from "drizzle-orm/node-postgres";
+import { Pool } from "pg";
+
+import { dbConfig } from "@connectors/resources/db/config";
+import { connectors } from "@connectors/resources/db/schema/connectors";
+
+import * as schema from "./schema";
+
+const pool = new Pool({
+  connectionString: dbConfig.getRequiredDatabaseURI(),
+});
+
+export function getDbConnection() {
+  const db = drizzle(pool, { schema });
+
+  return db;
+}
+
+const db = getDbConnection();
+
+// Retrieve the full object.
+const result = await db.select().from(connectors);
+
+// Retrieve only the selected field.
+const partialResult = await db
+  .select({ createdAt: connectors.createdAt })
+  .from(connectors)
+  .where(
+    or(eq(connectors.id, 42), eq(connectors.dataSourceName, "managed-notion"))
+  );
+
+// Fetch ConfluenceConfiguration with the connector.
+const confluenceConfigurationWithConnector =
```

Comment:
We won't support these types of relational queries. Engineers will be encouraged to use `.join()` as necessary.